### PR TITLE
*: sync more information links in translations

### PR DESCRIPTION
**Pull request by marchersimon**
This syncs all More information links in all translations with the new feature from #6405.
This is still a draft until #6538 is resolved.

### CODE REVIEW

**Comment by marchersimon** [comment]
This should be good to merge.

**Comment by marchersimon** [comment]
When you mean English, yes. Many pages redirect you to the right language automatically, and the rest gives you the option to change it manually.

File: pages.fr/common/matlab.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # matlab
 
 > Environnement de calcul numérique créé par MathWorks.
-> Plus d'informations : <https://fr.mathworks.com/help/matlab/>.
+> Plus d'informations : <https://uk.mathworks.com/help/matlab/>.
```
Comment (blueskyson) [inline pages.fr/common/matlab.md:4]:
```suggestion
> Plus d'informations : <https://fr.mathworks.com/help/matlab/>.
```

File: pages.it/windows/cmd.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # cmd
 
 > L'interprete dei comandi di Windows.
-> Maggiori informazioni: <https://docs.microsoft.com/it-it/windows-server/administration/windows-commands/cmd>.
+> Maggiori informazioni: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmd>.
```
Comment (blueskyson) [inline pages.it/windows/cmd.md:4]:
```suggestion
> Maggiori informazioni: <https://docs.microsoft.com/it-it/windows-server/administration/windows-commands/cmd>.
```

File: pages.ja/common/7z.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # 7z
 
 > 7-Zipは高圧縮率のファイルアーカイバ
-> 詳しくはこちら: <https://sevenzip.osdn.jp/>.
+> 詳しくはこちら: <https://www.7-zip.org>.
```
Comment (blueskyson) [inline pages.ja/common/7z.md:4]:
```suggestion
> 詳しくはこちら: <https://sevenzip.osdn.jp>.
```

File: pages.ja/common/7za.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 > 7-Zipは高圧縮率のファイルアーカイバ
 > より少ないアーカイブタイプをサポートする `7z`. スタンドアロンバージョン.
-> 詳しくはこちら: <https://sevenzip.osdn.jp/>.
+> 詳しくはこちら: <https://www.7-zip.org>.
```
Comment (blueskyson) [inline pages.ja/common/7za.md:5]:
```suggestion
> 詳しくはこちら: <https://sevenzip.osdn.jp>.
```

File: pages.ja/common/7zr.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 > 7-Zipは高圧縮率のファイルアーカイバ
 > .7zファイルをアーカイブタイプをサポートする
-> 詳しくはこちら: <https://sevenzip.osdn.jp/>.
+> 詳しくはこちら: <https://www.7-zip.org>.
```
Comment (blueskyson) [inline pages.ja/common/7zr.md:5]:
```suggestion
> 詳しくはこちら: <https://sevenzip.osdn.jp>.
```

File: pages.pt_BR/windows/cls.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # cls
 
 > Limpar a tela de saída.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/cls>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/cls>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/cls.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/cls>.
```

File: pages.pt_BR/windows/cmd.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # cmd
 
 > O interpretador de comandos do Windows.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/cmd>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmd>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/cmd.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/cmd>.
```

File: pages.pt_BR/windows/dir.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # dir
 
 > Listar os conteúdos de um diretório.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/dir>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/dir>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/dir.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/dir>.
```

File: pages.pt_BR/windows/mkdir.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # mkdir
 
 > Criar um diretório.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/mkdir>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/mkdir>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/mkdir.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/mkdir>.
```

File: pages.pt_BR/windows/print.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # print
 
 > Imprimir um arquivo de texto em uma impressora.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/print>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/print>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/print.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/print>.
```

File: pages.pt_BR/windows/type.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 # type
 
 > Mostrar o conteúdo de um arquivo.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/type>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/type>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/type.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/type>.
```

File: pages.pt_BR/windows/whoami.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # whoami
 
 > Mostra detalhes sobre o usuário atual.
-> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/whoami>.
+> Mais informações: <https://docs.microsoft.com/windows-server/administration/windows-commands/whoami>.
```
Comment (blueskyson) [inline pages.pt_BR/windows/whoami.md:4]:
```suggestion
> Mais informações: <https://docs.microsoft.com/pt-br/windows-server/administration/windows-commands/whoami>.
```

**Review by blueskyson — COMMENTED** [review]
Roughly reviewed all links. Just pick out all pages that use local-language links before sync. I think the sync option in set-information-link.py should be little bit modify: do not overwrite when the domain of link is same as that in English page.

**Comment by marchersimon** [comment]
@blueskyson but in #6538, we decided to have the exact same link in all pages.

**Review by blueskyson — APPROVED** [review]
OK.

File: pages.it/linux/mkfs.md [hunk 13]
```diff
@@ -1,7 +1,7 @@
 # mkfs
 
 > Costruisce un filesystem Linux su una partizione del disco rigido.
-> Questo comando è deprecato in favore degli strumenti specifici per filesystem: mkfs.<tipo>.
+> Maggiori informazioni: <https://manned.org/mkfs>.
```
Comment (navarroaxel) [inline pages.it/linux/mkfs.md:4]:
```suggestion
> Questo comando è deprecato in favore degli strumenti specifici per filesystem: mkfs.tipo.
> Maggiori informazioni: <https://manned.org/mkfs>.
```

File: pages.de/common/ansible-vault.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 # ansible-vault
 
 > Verschlüsselt und entschlüsselt Werte, Datenstrukturen und Dateien innerhalb von Ansible-Projekten.
-> Weitere Informationen: <https://docs.ansible.com/ansible/latest/user_guide/vault.html>.
+> Weitere Informationen: <https://docs.ansible.com/ansible/latest/user_guide/vault.html#id17>.
```
Comment (navarroaxel) [inline pages.de/common/ansible-vault.md:4]: Not sure about the `#id17`

**Review by navarroaxel — APPROVED** [review]

**Comment by blueskyson** [comment]
Thanks!

**Pull request merged**